libretro_core_options.h

```diff
@@ -911,7 +911,7 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(cpu_freq_scale),
       "CPU Frequency Scaling (Overclock)",
       NULL,
-      "Enable overclocking (or underclocking) of the emulated PSX CPU. Overclocking can eliminate slowdown and improve frame rates in certain games at the expense of increased performance requirements. Note that some games have an internal frame rate limiter and may not benefit from overclocking. May cause certain effects to animate faster than intended in some titles when overclocked.",
+      "Overclock (or underclock) the emulated PSX CPU. Overclocking can eliminate slowdown and improve frame rates in certain games at the expense of increased performance requirements. Note that some games have an internal frame rate limiter and may not benefit from overclocking. May cause certain effects to animate faster than intended in some titles when overclocked.",
```

Comment:
"eliminate slowdowns", "at the expense" is adequate?

Reply:
> "at the expense" is adequate?

What else do you suggest? "at the cost"?